blog/12-23-22- how-to-become-a-web3-developer/how-to-be-a-web3-developer.md

```diff
@@ -0,0 +1,187 @@
+---
+title: How to become a Web3 Developer
+description: There's an increasing need for more web3 developers as the internet gradually shifts from web2 to web3 technology. Learn how you can start your learning journey.
+authors:
+  - name: Elijah Sorinola
+    title: Content Writer, Celo Sage
+    image_url: blog/12-23-22- how-to-become-a-web3-developer/images/web3.png
```

Comment:
Kindly update this image_url link, there is a space in between

Reply:
hey, @izzecode some of the comments are still pending in particular, directories. Remove the spaces and Kindly update changes